prime-router/src/main/kotlin/azure/OpenAPIDocsFunction.kt

```diff
@@ -0,0 +1,95 @@
+package gov.cdc.prime.router.azure
+
+import com.azure.storage.blob.BlobClient
+import com.azure.storage.blob.BlobClientBuilder
+import com.microsoft.azure.functions.HttpMethod
+import com.microsoft.azure.functions.HttpRequestMessage
+import com.microsoft.azure.functions.HttpResponseMessage
+import com.microsoft.azure.functions.HttpStatus
+import com.microsoft.azure.functions.annotation.AuthorizationLevel
+import com.microsoft.azure.functions.annotation.BindingName
+import com.microsoft.azure.functions.annotation.FunctionName
+import com.microsoft.azure.functions.annotation.HttpTrigger
+import org.apache.logging.log4j.kotlin.Logging
+import java.io.ByteArrayOutputStream
+import java.io.InputStream
+import java.net.URL
+
+const val defaultConnEnvVar = "AzureWebJobsStorage"
+const val PARAM_RESOURCE_NAME = "resourceName"
+const val textJS = "text/javascript"
+const val textCSS = "text/css"
+const val textYaml = "text/yaml"
+
+/**
+ * Class for serving swagger UI page
+ */
+class OpenAPIDocsFunction : Logging {
+
+    @FunctionName("getApiDocs")
+    fun getAPIDocs(
```

Comment:
I'm wondering if there would be a way to use the [plugin](https://github.com/int128/gradle-swagger-generator-plugin#configure-swagger-ui) to set this up rather than having to maintain these files in src code

Reply:
worth some research...